exec/src/interfaces.ml

```diff
@@ -0,0 +1,57 @@
+(* FreeSpec
+ * Copyright (C) 2018–2019 ANSSI
+ *
+ * Contributors:
+ * 2019 Thomas Letan <thomas.letan@ssi.gouv.fr>
```

Comment:
I am not the contributor of this file, you are! :)

Reply:
Fixed in cdd7932.